pipelines/modules/platform/ocp.sh

```diff
@@ -0,0 +1,276 @@
+  log_info "Verifying CRDs"
+  oc get crd | grep "sonataflow" || log_warning "Sonataflow CRDs not found"
+  oc get crd | grep "knative" || log_warning "Serverless CRDs not found"
+  
+  log_success "Orchestrator infrastructure chart installed successfully"
+}
+
+# ============================================================================
+# Note: Tekton Pipelines functions moved to common.sh
+# Use install_tekton_pipelines_k8s from common.sh for Kubernetes clusters
+# For OpenShift, use install_pipelines_operator above
+# ============================================================================
+
+# ============================================================================
+# Cluster Setup Functions
+# ============================================================================
+
+# Setup OpenShift cluster for Helm deployments
+# Usage: cluster_setup_ocp_helm
+cluster_setup_ocp_helm() {
+  log_section "Setting up OpenShift Cluster for Helm Deployments"
+  
+  install_pipelines_operator
+  install_acm_ocp_operator
+  install_crunchy_postgres_ocp_operator
+
+  # Conditionally deploy orchestrator workflows
+  if [[ "${DEPLOY_ORCHESTRATOR_WORKFLOWS:-true}" == "true" ]]; then
+    install_orchestrator_infra_chart
+  else
+    log_info "Skipping orchestrator-infra chart deployment (DEPLOY_ORCHESTRATOR_WORKFLOWS=false)"
+  fi
+  
+  log_success "OpenShift cluster setup completed for Helm deployments"
+}
+
+# Setup OpenShift cluster for Operator deployments
+# Usage: cluster_setup_ocp_operator
+cluster_setup_ocp_operator() {
+  log_section "Setting up OpenShift Cluster for Operator Deployments"
+  
+  install_pipelines_operator
+  install_acm_ocp_operator
+  install_crunchy_postgres_ocp_operator
+  install_serverless_ocp_operator
+  install_serverless_logic_ocp_operator
+  
+  log_success "OpenShift cluster setup completed for Operator deployments"
+}
+
+# Setup Kubernetes cluster for Helm deployments
+# Usage: cluster_setup_k8s_helm
+cluster_setup_k8s_helm() {
+  log_section "Setting up Kubernetes Cluster for Helm Deployments"
+  
+  install_tekton_pipelines_k8s
+  # Note: OCM and Crunchy Postgres work with K8s but are disabled in values file
+  
+  log_success "Kubernetes cluster setup completed for Helm deployments"
+}
+
+# Setup Kubernetes cluster for Operator deployments
+# Usage: cluster_setup_k8s_operator
+cluster_setup_k8s_operator() {
+  log_section "Setting up Kubernetes Cluster for Operator Deployments"
+  
+  install_olm
+  install_tekton_pipelines_k8s
+  # Note: OCM and Crunchy Postgres work with K8s but are disabled in values file
+  
+  log_success "Kubernetes cluster setup completed for Operator deployments"
+}
+
+# ============================================================================
+# Platform Detection (using common.sh functions)
+# ============================================================================
+
+# Detect OpenShift platform (wrapper for common function)
+# Usage: detect_ocp
+detect_ocp() {
```

Comment:
The detection was not reliable, and it's now hardcoded, see: https://github.com/openshift/release/blob/3951d20775795d07b42e9f10d4df22d61af55182/ci-operator/step-registry/redhat-developer/rhdh/ocp/helm/redhat-developer-rhdh-ocp-helm-commands.sh#L51
Please delete this and the detect_platform_type function